lib/modules/platform/scm-manager/index.ts

```diff
@@ -0,0 +1,333 @@
+  }
+
+  if (!token) {
+    throw new Error('SCM-Manager API token not configured');
+  }
+
+  scmmClient = new ScmClient(endpoint, token);
+
+  const me = await scmmClient.getCurrentUser();
+  const gitAuthor = `${me.displayName} <${me.mail}>`;
+  const result = { endpoint, gitAuthor };
+
+  logger.info(`Plattform initialized ${JSON.stringify(result)}`);
+
+  return result;
+}
+
+export async function initRepo({
+  repository,
+  gitUrl,
+}: RepoParams): Promise<RepoResult> {
+  const repo = await scmmClient.getRepo(repository);
+  const defaultBranch = await scmmClient.getDefaultBranch(repo);
+  const url = getRepoUrl(
+    repo,
+    gitUrl,
+    /* istanbul ignore next */
+    hostRules.find({ hostType: id, url: scmmClient.getEndpoint() }).username ??
+      '',
+    process.env.RENOVATE_TOKEN ?? '',
+  );
+
+  config = {} as any;
+  config.repository = repository;
+  config.defaultBranch = defaultBranch;
+
+  await git.initRepo({
+    ...config,
+    url,
+  });
+
+  // Reset cached resources
+  invalidatePrCache();
+
+  const result = {
+    defaultBranch: config.defaultBranch,
+    isFork: false,
+    repoFingerprint: repoFingerprint(
+      config.repository,
+      scmmClient.getEndpoint(),
+    ),
+  };
+
+  logger.info(`Repo initialized: ${JSON.stringify(result)}`);
+
+  return result;
+}
+
+export async function getRepos(): Promise<string[]> {
+  const repos = (await scmmClient.getAllRepos()).filter(
+    (repo) => repo.type === 'git',
+  );
+  const result = repos.map((repo) => `${repo.namespace}/${repo.name}`);
+  logger.info(`Discovered ${repos.length} repos`);
+
+  return result;
+}
+
+export async function getBranchPr(branchName: string): Promise<Pr | null> {
+  return await findPr({ branchName, state: 'open' });
+}
+
+export async function findPr({
+  branchName,
+  prTitle,
+  state = 'all',
+}: FindPRConfig): Promise<Pr | null> {
+  const inProgressPrs = await getPrList();
+  const result = inProgressPrs.find(
+    (pr) =>
+      branchName === pr.sourceBranch &&
+      (!prTitle || prTitle === pr.title) &&
+      matchPrState(pr, state),
+  );
+
+  if (result) {
+    logger.info(`Found PR ${JSON.stringify(result)}`);
+    return result;
+  }
+
+  logger.debug(
+    `Could not find PR with source branch ${branchName} and title ${
+      prTitle ?? ''
+    } and state ${state}`,
+  );
+
+  return null;
+}
+
+export async function getPr(number: number): Promise<Pr | null> {
+  const inProgressPrs = await getPrList();
+  const cachedPr = inProgressPrs.find((pr) => pr.number === number);
+
+  if (cachedPr) {
+    logger.info(`Returning from cached PRs, ${JSON.stringify(cachedPr)}`);
+    return cachedPr;
+  }
+
+  try {
+    const result = await scmmClient.getRepoPr(config.repository, number);
+    logger.info(`Returning PR from API, ${JSON.stringify(result)}`);
+    return mapPrFromScmToRenovate(result);
+  } catch (error) {
+    logger.info(`Not found PR with id ${number}`);
```

Comment:
```suggestion
    logger.info(`Can not find a PR with id ${number}`);
```

The current error message is a bit wonky. How about this?